src/test/java/bio/terra/service/tabulardata/google/BigQueryPdaoTest.java

```diff
@@ -403,17 +405,65 @@ private void softDeleteRows(BigQueryProject bq,
     @Test
     public void testGetFullViews() throws Exception {
         Dataset dataset = readDataset("ingest-test-dataset.json");
-        String datasetBqDatasetName = bigQueryPdao.prefixName(dataset.getName());
 
-        BigQueryProject bigQueryProject = TestUtils.bigQueryProjectForDatasetName(
-            datasetDao, dataLocationService, dataset.getName());
+        // Stage tabular data for ingest.
+        String targetPath = "scratch/file" + UUID.randomUUID().toString() + "/";
+
+        String bucket = testConfig.getIngestbucket();
+
+        BlobInfo participantBlob = BlobInfo
+            .newBuilder(bucket, targetPath + "ingest-test-participant.json")
+            .build();
+        BlobInfo sampleBlob = BlobInfo
+            .newBuilder(bucket, targetPath + "ingest-test-sample.json")
+            .build();
+        BlobInfo fileBlob = BlobInfo
+            .newBuilder(bucket, targetPath + "ingest-test-file.json")
+            .build();
+
+        try {
+            bigQueryPdao.createDataset(dataset);
+
+            storage.create(participantBlob, readFile("ingest-test-participant.json"));
+            storage.create(sampleBlob, readFile("ingest-test-sample.json"));
+            storage.create(fileBlob, readFile("ingest-test-file.json"));
+
+            // Ingest staged data into the new dataset.
+            IngestRequestModel ingestRequest = new IngestRequestModel()
+                .format(IngestRequestModel.FormatEnum.JSON);
+
+            String datasetId = dataset.getId().toString();
+            connectedOperations.ingestTableSuccess(datasetId,
+                ingestRequest.table("participant").path(gsPath(participantBlob)));
+            connectedOperations.ingestTableSuccess(datasetId,
+                ingestRequest.table("sample").path(gsPath(sampleBlob)));
+            connectedOperations.ingestTableSuccess(datasetId,
+                ingestRequest.table("file").path(gsPath(fileBlob)));
 
-        DatasetTable participantTable = getTable(dataset, "participant");
-        DatasetTable sampleTable = getTable(dataset, "sample");
-        DatasetTable fileTable = getTable(dataset, "file");
-        List<DatasetTable> tables = Arrays.asList(participantTable, sampleTable, fileTable);
+            // Create a full-view snapshot!
+            DatasetSummaryModel datasetSummary = DatasetJsonConversion.datasetSummaryModelFromDatasetSummary(
+                dataset.getDatasetSummary());
+            SnapshotSummaryModel snapshotSummary = connectedOperations.createSnapshot(datasetSummary,
+                "snapshot-fullviews-test-snapshot.json", "");
+            SnapshotModel snapshot = connectedOperations.getSnapshot(snapshotSummary.getId());
 
-        bigQueryPdao.createSnaphotTableFromLiveViews(bigQueryProject, tables, datasetBqDatasetName);
```

Comment:
This method was returning a `String` (the result of generating SQL templates), not creating the BQ resources